addOns/paramdigger/src/main/resources/org/zaproxy/addon/paramdigger/resources/Messages.properties

```diff
@@ -55,12 +55,15 @@ paramdigger.dialog.cookieguess.wordlist.custom.file.location =  File Location:
 
 paramdigger.dialog.urlguess.chunksize = Number of Chunks:
 paramdigger.dialog.urlguess.methods = Methods:
-paramdigger.dialog.urlguess.methods.get = GET
-paramdigger.dialog.urlguess.methods.post = POST
-paramdigger.dialog.urlguess.methods.xml = XML
-paramdigger.dialog.urlguess.methods.json = JSON
+paramdigger.dialog.methods.get = GET
+paramdigger.dialog.methods.post = POST
+paramdigger.dialog.methods.xml = XML
+paramdigger.dialog.methods.json = JSON
 paramdigger.dialog.urlguess.methods.tooltip = Multi-select with CTRL + Click (or CMD + Click).
 
+paramdigger.dialog.headerguess.methods = Methods: 
+paramdigger.dialog.headerguess.methods.tooltip = Multi-select with CTRL + Click (or CMD + Click).
```

Comment:
These could be made generic as well.

Reply:
Generic?

Reply:
This message and the one two lines above could use the same key/value if the name was just made generic.